### DISCUSSION TITLE
[Feature Request]: 极空间极影视的支持

### DISCUSSION THREAD

**Discussion by dalanchongjun** [jxxghp/MoviePilot#3347] [讨论]
### 当前程序版本

1.8.0

### 运行环境

Docker

### 功能改进类型

主程序

### 功能改进

大佬 能不能研究一下 与极空间的极影视 联动，我用CD2把115网盘映射到nas，下载的资源我是直接整理入115网盘，入库之后，极影视要手动重新获取一次，才能够看到最新入库的视频，大佬能不能看看怎么处理一下？

**Comment by behinder85** [comment]
但这个是极空间不能定时刷新的原因吧，和MP有关系？（存疑

**Comment by dalanchongjun** [comment]
就是MP能调用极影视 让极影视刷新列表么

**Comment by behinder85** [comment]
> 就是MP能调用极影视 让极影视刷新列表么

个人觉得够呛，首先作者有无极空间另说，其次这个涉及极空间的权限管理了吧，MP应该没那么高权限的

**Comment by cddjr** [comment]
至少需要极空间的本地访问账号及密码才有可能实现，而且也不适合做进mp内，只能尝试用插件实现。

**Comment by InfinityPacer** [comment]
极空间是类似Plex的局部刷新的方案，系统层面上做了文件变更的监控，对于网盘的对接，应该给极空间提需求
